source/sorted_table_model.ts

```diff
@@ -103,16 +128,31 @@ export class SortedTableModel extends TableModel {
 
   public connect(slot: (operations: Operation[]) => void):
       Kola.Listener<Operation[]> {
-    return null;
+    return this.dispatcher.listen(slot);
+  }
+
+  private handleOperations(newOperations: Operation[]): void {
+    this.beginTransaction();
+    for(const operation of newOperations) {
+      if(operation instanceof AddRowOperation) {
+        this.rowAdded(operation);
+      } else if(operation instanceof RemoveRowOperation) {
+        this.beginTransaction();
```

Comment:
No need to begin a new transaction here, a transaction was already started on line 135.